app/src/main/res/layout/fragment_notification.xml

```diff
@@ -8,4 +8,13 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:id="@+id/rv_notifications"/>
+    <TextView
+        android:id="@+id/tv_no_notification_msg"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:gravity="center"
+        android:visibility="gone"
+        android:text="No Notifications"
```

Comment:
@miPlodder , Don't use raw strings instead create a resource in strings.xml for this.

Reply:
I will make the changes and update the PR.